README.md

```diff
@@ -74,6 +58,22 @@ Most common commands for all projects:
 setup back to development mode, you must run `./gradlew vaadinPrepareFrontend`
 with the `productionMode` effectively set to false (e.g. by ommitting the `-Pvaadin.productionMode` flag).
 
+There are the following tasks:
```

Comment:
```suggestion
All tasks provided by the plugin:
```

This list feels bit weird with the current title as it kind of repeats what is already discussed above.  Maybe the suggested wording or something else would let readers know what the list below is.